src/main/java/org/takes/facets/auth/social/PsTwitter.java

```diff
@@ -136,19 +184,4 @@ private String token()
             .as(JsonResponse.class)
             .json().readObject().getString("access_token");
     }
-
-    /**
-     * Make identity from JSON object.
-     * @param json JSON received from Twitter
-     * @return Identity found
-     */
-    private static Identity parse(final JsonObject json) {
```

Comment:
@adko-pl 
1. In this case the fact that it is `static` is not a problem. It is not `public` and therefore not a utility class. Just an internal private method.
2. Note that now the code looks less readable (where you used to call `parse`). let's revert this for the sake of readability.